syntax = "proto3";

package demo;

import "demo/demo_messages.proto";

service DemoService {
  rpc SayHello (HelloRequest) returns (HelloReply);
}
